src/vos/vos_gc.c

```diff
@@ -130,7 +130,7 @@ gc_drain_evt(struct vos_gc *gc, struct vos_pool *pool, daos_handle_t coh,
 
 	D_DEBUG(DB_TRACE, "drain %s evtree, creds=%d\n", gc->gc_name, *credits);
 	rc = evt_drain(toh, credits, empty);
-	evt_close(toh);
+	D_ASSERT(evt_close(toh) == 0);
```

Comment:
Suffices for coverity because evt_close would never be called at all on the condition that the return code is non-zero (evt_open would have failed).